src/Modal.svelte

```diff
@@ -34,7 +36,6 @@
   export let backdropClassName = '';
   export let contentClassName = '';
   export let external = undefined;
-  export let fade = true;
```

Comment:
We were aligning with Reactstrap's approach that sets fade on/off via a boolean prop (https://reactstrap.github.io/components/modals/).
Could you keep this behavior and add the duration props?

Reply:
Watch now. Do you like?